src/Confluent.SchemaRegistry.Rules/Confluent.SchemaRegistry.Rules.csproj

```diff
@@ -16,7 +16,7 @@
         <Title>Confluent.SchemaRegistry.Rules</Title>
         <AssemblyName>Confluent.SchemaRegistry.Rules</AssemblyName>
         <VersionPrefix>2.11.1</VersionPrefix>
-        <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
+        <TargetFrameworks>netstandard2.1;net462;net6.0;net8.0</TargetFrameworks>
```

Comment:
Curious why net462 is added as one of the targets - is it some request from customers?